RevenueCatUI/CustomerCenter/Views/WrongPlatformView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  WrongPlatformView.swift
+//
+//
+//  Created by Andrés Boedo on 5/3/24.
+//
+
+import Foundation
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct WrongPlatformView: View {
+
+    @State
+    private var store: Store?
+
+    @Environment(\.openURL)
+    private var openURL
+
+    init() {
+        self._store = State(initialValue: nil)
+    }
+
+    init(store: Store) {
```

Comment:
If only used for previews/tests.
```suggestion
    fileprivate init(store: Store) {
```